windows/check_quiesce_snapshot/check_quiescing_type.yml

```diff
@@ -0,0 +1,91 @@
+# Copyright 2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+- name: "Initialize not support application consistent quiescing type"
+  ansible.builtin.set_fact:
+    winsrv_app_quiesce_support: false
+
```

Comment:
This var is only used from line 65. I think either this set_fact can be moved before line 65 or move line 65 ~ 70 after this task. After all, the tasks between them has nothing to do with `winsrv_app_quiesce_support` setting.

Reply:
I prefer to put variables initialization in the front of the file, it's more readable and clean.